### PR TITLE
Relnotes for Desktop Edge 2.1.6.0

### PR DESCRIPTION
**Pull request by usha-mandya**
Release notes for Docker Desktop Edge 2.1.6.0 release

### CODE REVIEW

File: docker-for-mac/edge-release-notes.md [hunk 1]
```diff
@@ -58,6 +77,7 @@ Fixed an issue that caused VMs running on older hardware with macOS Catalina to
 - Improved the navigation in **Settings** and **Troubleshoot** UI.
 - Fixed a bug in the UEFI boot menu that sometimes caused Docker Desktop to hang during restart. Fixes [docker/for-mac#2655](https://github.com/docker/for-mac/issues/2655) and [docker/for-mac#3921](https://github.com/docker/for-mac/issues/3921).
 - Docker Desktop now allows users to access the host’s SSH agent inside containers. Fixes [docker/for-mac#410](https://github.com/docker/for-mac/issues/410)
+- Docker Machine is not included in the Docker Desktop installer anymore. You can download it separately from the [Docker Machine releases](https://github.com/docker/machine/releases) page.
```
Comment (traci-morrison) [inline docker-for-mac/edge-release-notes.md]: Suggest: Docker Machine is no longer included in the Docker Desktop installer.

File: docker-for-windows/edge-release-notes.md [hunk 2]
```diff
@@ -64,6 +92,7 @@ This release contains a Kubernetes upgrade. Note that your local Kubernetes clus
 
 - Improved the navigation in **Settings** and **Troubleshoot** UI.
 - Fixed a bug that prevented users from accessing WSL 2 Tech Preview. Fixes [docker/for-win#4734](https://github.com/docker/for-win/issues/4734).
+- Docker Machine is not included in the Docker Desktop installer anymore. You can download it separately from the [Docker Machine releases](https://github.com/docker/machine/releases) page.
```
Comment (traci-morrison) [inline docker-for-windows/edge-release-notes.md]: Suggest: Docker Machine is no longer included in the Docker Desktop installer.

File: docker-for-mac/edge-release-notes.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-mac/edge-release-notes.md]: Fixed.

File: docker-for-windows/edge-release-notes.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-windows/edge-release-notes.md]: Fixed.

**Pull request merged**